packages/validator/src/services/indices.ts

```diff
@@ -76,6 +76,14 @@ export class IndicesService {
   private async pollValidatorIndicesInternal(pubkeysHex: PubkeyHex[]): Promise<ValidatorIndex[]> {
     const pubkeysHexToDiscover = pubkeysHex.filter((pubkey) => !this.pubkey2index.has(pubkey));
 
+    pubkeysHex.forEach((pubkey) => {
+      if (this.pubkey2index.has(pubkey)) {
+        this.logger.info("Validator status: ", {status: "active"});
```

Comment:
I think this should be a one liner, having multiple lines is not that easy to read in my opinion, also it might happen that they are not grouped together, generally can't rely on the order of logs as they are written asynchronously

```suggestion
        this.logger.info("Validator", {index: this.pubkey2index.get(pubkey), status: "active", pubkey}});
```